src/main.cpp

```diff
@@ -41,6 +41,11 @@ int main()
     Keyboard keyboard;
     sf::Clock updateClock;
 
+    //gui bools
+    bool m_profilerOpen= false;
+    bool m_infoOpen = false;
+    bool m_brushOpen = false;
```

Comment:
`m_` prefix is only used for class members. However, the bar is not needed (for anyways, wasn't planning to have this epc. for the entire game) 

An alternative could be pressing a key eg ` or `F3` to open the debug/profiling menus